src/core/utils.js

```diff
@@ -804,6 +805,27 @@ export function children(element, selector) {
   }
 }
 
+/**
+ * Calculates indentation when the element starts after a newline.
+ * The value will be empty if no newline or any non-whitespace exists after one.
+ * @param {Element} element
+ */
+export function getElementIndentation(element) {
```

Comment:
@saschanaz I'm failing to understand this function. I think the comment can be improved.
Adding some unit tests would be really nice.

Reply:
Ah. Makes sense now. Thanks!